### DISCUSSION TITLE
I can't define jaws' opening using field size and SSD

### DISCUSSION THREAD

**Discussion by phantasuson** [nrc-cnrc/EGSnrc#803] [Questions] [answered]
![image](https://user-images.githubusercontent.com/80371548/144355513-42cad2b0-a9bd-4b7c-a3e5-fcdb2f24dc35.png)

**Comment by phantasuson** [comment, marked as answer]
detail：
```
bad window path name ".syncjaws6.child"
bad window path name ".syncjaws6.child"
    while executing
"toplevel .syncjaws$id.child.qkdef"
    (procedure "quikdyndef" line 9)
    invoked from within
"quikdyndef 6"
    invoked from within
".dynjaws6.child.f.quikdyndef invoke"
    ("uplevel" body line 1)
    invoked from within
"uplevel #0 [list $w invoke]"
    (procedure "tk::ButtonUp" line 24)
    invoked from within
"tk::ButtonUp .dynjaws6.child.f.quikdyndef"
    (command bound to event)
```

**Reply by ftessier** [reply]
Thanks for the question! @blakewalters, does this Tk error iommediately ring a bell?

**Reply by rtownson** [reply]
This is a bug, I have reproduced it! You can see in the error that it is trying to reference syncjaws from within a dynjaws CM, for some reason.

**Reply by ftessier** [reply]
All right thanks, I'll let you post this as an issue!

**Reply by rtownson** [reply]
See the fix here: https://github.com/nrc-cnrc/EGSnrc/pull/806
